ts/features/wallet/component/FeaturedCardCarousel.tsx

```diff
@@ -118,7 +117,7 @@ const mapStateToProps = (state: GlobalState) => ({
   bpdActiveBonus: bpdEnabledSelector(state),
   // FIXME replace with Selector when the API implementation is completed.
   cgnActiveBonus: false,
-  availableBonusesList: pot.getOrElse(availableBonusTypesSelector(state), [])
+  availableBonusesList: visibleAvailableBonusSelector(state)
```

Comment:
Since we are using `visibleAvailableBonusSelector` that combine the remote data with local FFs, we can remove the local usage of FF


### from
```typescript
const anyBonusNotActive =
    (!pot.getOrElse(props.bpdActiveBonus, false) && bpdEnabled) ||
    (!props.cgnActiveBonus && cgnEnabled);
```

### to
```typescript
const hasBpdActive: boolean | undefined = pot.isSome(props.bpdActiveBonus)
    ? props.bpdActiveBonus.value
    : undefined;
  const anyBonusNotActive = hasBpdActive === false || !props.cgnActiveBonus;
```

This code also solves a bad condition that displays an item when the data is loading
`pot.getOrElse(props.bpdActiveBonus, false)` -> this is `false` (the item is displayed) even when the relative data is loading or in error. 
This should solve
```typescript
const hasBpdActive: boolean | undefined = pot.isSome(props.bpdActiveBonus)
    ? props.bpdActiveBonus.value
    : undefined;
```

- not available data: `hasBpdActive = undefined`
- available data: `hasBpdActive = true/false`

What do you think?

Reply:
I didn't focused previously on rendering condition of the items since the focus of the PR was to replace the badly used selector, but you're right we can remove FFs logic from component since is now prevented by the specific selector

Anyway Yep! I found it a good solution